portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+function getQuote() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('quote-container').innerText = quote;
+  });
+  console.log(response)
```

Comment:
I'm not entirely familiar with JS, does this work? response and quote won't be defined when this runs

Reply:
This works! 'response' and 'quote' are just for readability.